allchblk.c

```diff
@@ -984,6 +989,34 @@ GC_INNER void GC_freehblk(struct hblk *hbp)
           hhdr = prevhdr;
         }
       }
+    #else  /* !defined(__CHERI_PURE_CAPABILITY__) */
+    /* Coalesce with successor, if possible */
+      if(0 != nexthdr && HBLK_IS_FREE(nexthdr) && IS_MAPPED(nexthdr)
+         && (signed_word)(hhdr -> hb_sz + nexthdr -> hb_sz) > 0
+	 && (cheri_base_get(hbp) <= cheri_address_get(next))
```

Comment:
Weird indendation?

Reply:
Fixed in [632aa0c](https://github.com/capablevms/bdwgc/pull/8/commits/632aa0c72a478ccf9fc201842578f7d35b822166).